core/src/main/resources/lib/form/hetero-list.jelly

```diff
@@ -112,11 +116,11 @@ THE SOFTWARE.
   <j:set var="targetType" value="${attrs.targetType?:it.class}"/>
   <div class="hetero-list-container ${hasHeader?'with-drag-drop':''} ${attrs.oneEach?'one-each':''} ${attrs.honorOrder?'honor-order':''}">
     <!-- display existing items -->
-    <j:forEach var="i" items="${attrs.items}">
+    <j:forEach var="i" items="${attrs.items}"><!-- TODO consider customizedFields: how to distinguish default items from user-configured items? -->
```

Comment:
Does this TODO relate to an already existing issue or one introduced by this PR?

Reply:
This is a long outstanding design issue with this tag.

Reply:
Looks like we can also resolve this one.